node/node.go

```diff
@@ -522,6 +482,14 @@ func makeSeedNode(cfg *config.Config,
 
 // OnStart starts the Node. It implements service.Service.
 func (n *nodeImpl) OnStart() error {
+
+	if n.config.RPC.PprofListenAddress != "" {
+		go func() {
+			logger.Info("Starting pprof server", "laddr", n.config.RPC.PprofListenAddress)
+			logger.Error("pprof server error", "err", http.ListenAndServe(n.config.RPC.PprofListenAddress, nil))
+		}()
+	}
```

Comment:
I like this change 👍 Are we able to turn it off `OnStop()`?

Reply:
I don't believe that's a thing that's currently supported, we certainly never have before.